setup.sh

```diff
@@ -100,58 +111,94 @@ check() {
         exit 1
     fi
 
+    # setup environment file
     if [ ! -f "_env" ]; then
-        echo "Creating a configuration file..."
-        cp _env.example _env
+        echo '# Environment variables for for WordPress site' > _env
+        echo '# please include the scheme http:// or https:// in the URL variable' >> _env
+
+        echo 'WORDPRESS_URL=http://'wordpress.svc.${TRITON_ACCOUNT}.${TRITON_DC}.cns.triton.zone >> _env
+        echo 'WORDPRESS_SITE_TITLE=Autopilot Pattern WordPress test site' >> _env
+        echo 'WORDPRESS_ADMIN_EMAIL=user@example.net' >> _env
+        echo 'WORDPRESS_ADMIN_USER=username' >> _env
+        echo 'WORDPRESS_ADMIN_PASSWORD='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 11) >> _env
+        echo 'WORDPRESS_ACTIVE_THEME=theme' >> _env
+        echo 'WORDPRESS_CACHE_KEY_SALT='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 53) >> _env
+        echo '#WORDPRESS_TEST_DATA=false # uncomment to import a collection of test content on start' >> _env
+        echo >> _env
+
+        echo '# Wordpress security salts' >> _env
+        echo '# These must be unique for your install to ensure the security of the site' >> _env
+        echo 'WORDPRESS_AUTH_KEY='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 53) >> _env
+        echo 'WORDPRESS_SECURE_AUTH_KEY='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 53) >> _env
+        echo 'WORDPRESS_LOGGED_IN_KEY='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 53) >> _env
+        echo 'WORDPRESS_NONCE_KEY='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 53) >> _env
+        echo 'WORDPRESS_AUTH_SALT='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 53) >> _env
+        echo 'WORDPRESS_SECURE_AUTH_SALT='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 53) >> _env
+        echo 'WORDPRESS_LOGGED_IN_SALT='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 53) >> _env
+        echo 'WORDPRESS_NONCE_SALT='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 53) >> _env
+        echo >> _env
+
+        echo '# Environment variables for MySQL service' >> _env
+        echo '# WordPress database/WPDB information' >> _env
+        echo 'MYSQL_USER=wpdbuser' >> _env
+        echo 'MYSQL_PASSWORD='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 7) >> _env
+        echo 'MYSQL_DATABASE=wp' >> _env
+        echo '# MySQL replication user, should be different from above' >> _env
+        echo 'MYSQL_REPL_USER=repluser' >> _env
+        echo 'MYSQL_REPL_PASSWORD='$(cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | head -c 7) >> _env
         echo >> _env
+
+        echo '# Environment variables for backups to Manta' >> _env
+        echo 'MANTA_BUCKET= # an existing Manta bucket' >> _env
+        echo 'MANTA_USER= # a user with access to that bucket' >> _env
+        echo 'MANTA_SUBUSER=' >> _env
+        echo 'MANTA_ROLE=' >> _env
+        echo 'MANTA_URL=https://us-east.manta.joyent.com' >> _env
```

Comment:
also, do we want to use ${TRITON_DC} in the MANTA_URL here to save the backups into manta in the same DC we are using for the containers?